bin/generate

```diff
@@ -48,10 +48,8 @@ parser.on('--verify', 'Verify all exercises') do
       Generator.new(exercise).generate(f.path)
       generated_code = f.read
       File.delete(f.path)
-      fail VerificationError unless current_code == generated_code
+      fail VerificationError.new(exercise) unless current_code == generated_code
```

Comment:
Just to note:

```suggestion
      fail VerificationError, exercise unless current_code == generated_code
```

That would be identical behavior, and I believe how you generally see this constructed.